receiver/purefareceiver/config.go

```diff
@@ -64,6 +66,23 @@ type ReloadIntervals struct {
 }
 
 func (c *Config) Validate() error {
-	// TODO(dgoscn): perform config validation
-	return nil
+	var errs error
+
+	if c.Settings.ReloadIntervals.Array.String() == "" {
```

Comment:
This seems to have been reverted again. Please, use what's being used for purefb.

Reply:
Sure @jpkrohling